site/docs/components/dropdown/usage.mdx

```diff
@@ -0,0 +1,47 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Using the Dropdown component
+
+#### When to use Dropdown
+
+- When a user needs the ability to choose one value from a set of five to 10 options.
+- When only the selected value from a set of options needs to be visible once the selection is made.
+
+#### When not to use Dropdown
+
+- When you have more than 10 options and users would benefit from filtering to narrow down the available options. Instead, use [Combo Box](../combo-box).
+- When you have fewer than five options and need to one option to be selected. Instead, use [Radio Button Group](../button).
+- When you have fewer than five options and multiple selections are required, or possible. Instead, use [Checkbox Group](../checkbox).
+- If you have a Boolean selection, for instance “on” or “off”. Instead, use [Switch](../switch).
+- When you have large number (>100) of list items, to prevent performance issues without virtualisation, we strongly recommend adding a filter e.g. use a [Combo Box](../combo-box) instead of Dropdown
```

Comment:
should this be the American `virtualization` ?

```suggestion
- When you have large number (>100) of list items, to prevent performance issues without virtualization, we strongly recommend adding a filter e.g. use a [Combo Box](../combo-box) instead of Dropdown
```